<!DOCTYPE HTML>
<!--
	Dimension by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Phillip A. Richmond</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<div class="logo">
							<span class="icon fa-gem"></span>
						</div>
						<div class="content">
							<div class="inner">
								<h1>Phillip A. Richmond</h1>
								<p>A precision health scientist, educator, and consultant. <br />
								Follow the science wherever it leads you. </p>
							</div>
						</div>
						<nav>
							<ul>
								<li><a href="#about">About Me</a></li>
								<li><a href="#teaching">Teaching</a></li>
								<li><a href="#publications">Publications</a></li>
								<li><a href="#consulting">Consulting</a></li>
								<li><a href="https://www.linkedin.com/in/phillip-andrew-richmond-phd-43838420/">LinkedIn</a></li>
								<li><a href="#contact">Contact</a></li>
								<!--<li><a href="#elements">Elements</a></li>-->
							</ul>
						</nav>
					</header>

				<!-- Main -->
					<div id="main">

						<!-- Intro -->
							<article id="about">
								<h2 class="major">About Me</h2>
								<span class="image main"><img src="images/headshot.jpg" alt="" /></span>
								<p>My name is Dr. Phillip Richmond, and I have a passion for precision health research. I was first exposed to high throughput sequencing and genomics in 2010 as an undergraduate researcher in Robin Dowell's lab, and I immediately knew I had found my calling in the world of genomics and bioinformatics. Fast forward to today, and my skills in bioinformatics lead to clinical answers, and in some cases life changing discoveries, for patients and their families. </p>
								<p>In my free time you will find me outside with my dogs, Sherlock and Rosie.</p>
								<h3>Education</h3>
								<p><b>2015-2020:</b> PhD in Bioinformatics from the University of British Columbia, under supervision of Dr. Wyeth Wasserman. </p>
									My thesis research focused on expanding the utility of whole genome sequencing in the diagnosis of rare genetic diseases, and you can read my thesis <a href="https://open.library.ubc.ca/cIRcle/collections/ubctheses/24/items/1.0394775">here</a>. 
									During my thesis I investigated unsolved rare genetic disease cases, discovered a novel tandem repeat expansion, helped to build tools for variant calling and simulation, and investigated candidate modifying markers leading to variable phenotypes seen in X-linked Adrenoleukodystrophy. </p>
									<p><b>2008-2012:</b> BA in Molecular, Cellular, & Developmental Biology <i>summa cum laude</i> from the University of Colorado, Boulder.</p>  
									<p>My undergraduate thesis work focused on identifying the genotypic effects of polyploidy on the adaptation of saccharomyces cerevisiae when subjected to metabolic stress. </p>
								<h3>Research Experience</h3>
									<p><b>2020-Current:</b> Staff Scientist, BC Children's Hospital Research Institute.</p> 
									<p> In this role I am developing the <a href="http://bcchr.ca/phi">Precision Health Initiative</a>, under supervision of Dr. Stuart Turvey. In this role I focus on three main objectives including 1) Coordinating computational infrastructure solutions for researchers at BCCHR, 2) Teaching clinical and research audiences about precision health technology including hands-on exposure to data analysis, and 3) Analysis of unsolved rare genetic disease cases within the BCCHR <a href="http:/bcchr.ca/thehub">Rare Disease Discovery Hub</a>, typically with the use of multi-omics approaches.</p>
									<p><b>2012-2015:</b> Professional Research Assistant, University of Colorado, Boulder</p>
									<p>In this role I supported diverse projects spanning model organism genomics, and led training initiatives at the Biofrontier's Institute, under supervision of Dr. Robin Dowell-Deen. Projects included regulatory genomic studies in yeast, directed evolution studies in yeast, behavioural genetics of alcoholism in mice, and histone de-acetylase cancer therapeutic epigenome profiling in tumor-derived cell lines.</p> 
								<h3>Publications</h3>	
								<p>To read more about my published works, head over to the <a href="#Publications">Publications</a> page, or visit my <a href="https://scholar.google.ca/citations?user=Q5vulG4AAAAJ&hl=en">Google Scholar</a>. </p>
								</article>

						<!-- Teaching -->
							<article id="teaching">
								<h2 class="major">Teaching</h2>
								<span class="image main"><img src="images/Sherlock_sunny.jpg" alt="" /></span>
								<p>Under construction, May 2022</p>
								<h2 class="minor">Clinical</h2>
								<p><b>BC Children's and Women's Hospital Medical Genetics Resident Academic Half-day. ”Hands-on Introduction to Rare Disease Genomics”. December 10th, 2021.</b></p>
								<p><b>Description:</b> This hands-on activity covers a brief review of exome sequencing for the diagnosis of rare genetic diseases, followed by a hands-on activity where participants will work through a mock rare disease case and present their candidate gene in a clinical round table format. The ses- sion concludes with a lecture on next steps for ”negative” exome reports.</p>
								<p><b>Attendees:</b> 5</p>
								<p><b>Delivery:</b> 30 min lecture, 1 hour activity, 45 min round-table presentations, 45 min lecture.</p>
								<p></p>
								<p><b>UBC Master's in Genetic Counselling Training Program plus 3rd year Genetics Resident. ”Genome Analysis Module”. November 1st-19th, 2021.</b></p>
								<p><b>Description:</b> This is the second version of a hands-on module teaching genetic counselling students how to go from raw data to a diagnosis in the context of rare disease genomics. Students are assigned a case with raw sequencing data, and they learn Linux, high performance computing, short-read mapping/variant calling/visualization, and gene prioritization. They then present their candidate gene in a clinical round table format. The hands-on nature of this module relies upon access to high performance computing, herein provided by BC Children's Hospital Research Institute.</p>
								<p><b>Attendees:</b> 7</p>
								<p><b>Delivery:</b> Nine 2-hour sessions, in person.</p>
								<p><b>Content:</b> <a href="https://phillip-a-richmond.github.io/GenomeAnalysisModule/">https://phillip-a-richmond.github.io/GenomeAnalysisModule/</a></p>
								<p></p>
								<p><b>Canadian Allergy and Clinical Immunology Distributed half-day. ”Hands-on Introduction to Rare Disease Genomics”. September 1st, 2021.</b></p>
								<p><b>Description:</b> This is a hands-on activity that starts with a lecture about the basics for rare disease genomic analysis (described above). Then the participants split into groups of 10 and get assigned a case with an Exomiser Output HTML file that they then download, and navigate to identify the causal variant/gene for their case. In the last hour, participants will present their top 2 candidate genes to the rest of the group, with the goal of unanimously agreeing on the diagnosis for each case.</p>
								<p><b>Attendees:</b> 40</p>
								<p><b>Delivery:</b> 1-hour lecture, 1-hour activity, 1-hour group presentations. Online via Zoom</p>
								<p></p>
								<p><b>BC Children's and Women's Hospital Paediatric Sub-specialty Academic Half-day. ”Introduction to Rare Disease Genomics”. June 4th, 2021.</b></p>
								<p><b>Description:</b> Lecture about the basics for rare disease genomic analysis, including how the data is processed and presented/communicated, the databases available online for deeper investigations of candidate genes/variants, and some options for what you can do next when there is a ”variant of uncertain significance” as the only reported variant for a case.</p>
								<p><b>Attendees:</b> 35</p>
								<p><b>Content:</b> 1-hour lecture. Online via Zoom.</p>
								<p><b>UBC Master's in Genetic Counselling Training Program plus 3rd year Genetics Resident. ”Genome Analysis Module”. November 4th-18th, 2020.</b></p>
								<p><b>Description:</b> This is the first version of a hands-on module teaching genetic counselling students how to go from raw data to a diagnosis in the context of rare disease genomics. Students are assigned a case with raw sequencing data, and they learn Linux, high performance computing, short-read mapping/variant calling/visualization, and gene prioritization. They then present their candidate gene in a clinical round table format. The hands-on nature of this module relies upon access to high performance computing, herein provided by BC Children's Hospital Research Institute.</p>
								<p><b>Attendees:</b> 6</p>
								<p><b>Delivery:</b> Five 4-hour sessions. Hybrid in-person and online.</p>
								<p><b>Content:</b> <a href="https://phillip-a-richmond.github.io/GenomeAnalysisModule/">https://phillip-a-richmond.github.io/GenomeAnalysisModule/</a></p>


							</article>

						<!-- Publications -->
							<article id="publications">
								<h1 class="major">Publications</h1>
								<span class="image main"><img src="images/sherlock_tree.jpg" alt="" /></span>
								<p>This is a selection of some of my favourite publications where I played a major/lead role, split into 3 categories below.</p>
								<p>For a full list of publications, please visit my <a href="https://scholar.google.ca/citations?user=Q5vulG4AAAAJ&hl=en">Google Scholar</a></p>
								<h2 class="minor">Medical Genetics, Genomics, and Rare Disease</h2>
								<span class="image main"><img src="images/GLS_Titleshot.png" alt="" /></span>
								<p><b>Glutaminase Deficiency caused by Short Tandem Repeat Expansion in GLS (co-first)</b></p>
								<p>This was a collaborative discovery involving biochemical experts, medical geneticists, molecular biologists, and bioinformaticians (most importantly of course). We found this tandem repeat expansion to be causal for repression of GLS expression, leading to a multitude of neurodevelopmental and movement-based phenotypes.</p>
								<p>Andre BP van Kuilenburg*, Maja Tarailo-Graovac*, <b>Phillip A Richmond*</b>, Britt I Drogemoller, Mahmoud A Pouladi, Rene Leen, Koroboshka Brand-Arzamendi, Doreen Dobritzsch, Egor Dolzhenko, Michael A Eberle, Bruce Hayward, Meaghan J Jones, Farhad Karbassi, Michael S Kobor, Janet Koster, Daman Kumari, Meng Li, Julia MacIsaac, Cassandra McDonald, Judith Meijer, Charlotte Nguyen, Indhu-Shree Rajan-Babu, Stephen W Scherer, Bernice Sim, Brett Trost, Laura A Tseng, Marjolein Turkenburg, Joke JFA van Vugt, Jan H Veldink, Jagdeep S Walia, Youdong Wang, Michel van Weeghel, Galen EB Wright, Xiaohong Xu, Ryan KC Yuen, Jinqiu Zhang, Colin J Ross, Wyeth W Wasserman, Michael T Geraghty, Saikat Santra, Ronald JA Wanders***, Xiao-Yan Wen***, Hans
									R Waterham***, Karen Usdin***, Clara DM van Karnebeek***. (2019), ”Glutaminase Deficiency Caused by Short Tandem Repeat Expansion in GLS”, New England Journal of Medicine 380(15) 1433-1441 (2020). (online at <a href="https://doi.org/10.1056/NEJMoa1806627">https://doi.org/10.1056/NEJMoa1806627</a>). *Co-first author, ***Co- last author.</p>
								<span class="image main"><img src="images/XALD_Titleshot.png" alt="" /></span>
								<p><b>Multi-omic approach to identify phenotypic modifiers underlying Cerebral Demyelination in X-linked adrenoleukodystrophy (co-first)</b></p>
								<p>This was a tour de force, a project where I was responsible for the WGS analysis, but ended up writing a story about candidate signals in the data, and a lack of power (despite having brother-pairs) for multi-omic discovery of biomarkers. Lessons were learned, and larger datasets are needed, but this can serve as a dataset for examining signals identified in independent projects.</p>
								<p><b>Phillip A Richmond*</b>, Frans van der Kloet*, Frederic M. Vaz, Antoine H.C. van Kampen, An- uli Uzozie, Philipp F. Lange, David Lin, Michael Kobor, Emma Graham, Sara Mostafavi, Perry Moerland, Wyeth W Wasserman***, Marc Engelen***, Stephan Kemp***, Clara van Karnebeek***. ”Multi-omic approach to identify phenotypic modifiers undelrying cerebral demyelination in X- linked adrenoleukodystrophy.” Frontiers in Cellular and Developmental Biology, 8, 520. (2020). (on- line at <a href="https://doi.org/10.1101/2020.03.19.20035063">https://doi.org/10.1101/2020.03.19.20035063</a>). *Co-first author, ***Co-last author.</p>
								<span class="image main"><img src="images/VariabilityConundrum_Titleshot.png" alt="" /></span>
								<p><b>The variability conundrum in neurometabolic degenrative diseases (co-first)</b></p>
								<p>This article is a perspective commentary, following on the publication of our multi-omic study to resolve modifiers of X-linked Adrenoleukodystrophy (xALD).</p>
								<p>Clara van Karnebeek*, <b>Phillip A. Richmond*</b>, Frans van der Kloet, Wyeth W. Wasserman, Marc Engelen, Stephan Kemp. ”The variability conundrum in neurometabolic degenerative diseases.” Molecular Genetics and Metabolism, (2020). (online at <a href="https://doi.org/10.1016/j.ymgme.2020.11.002">https://doi.org/10.1016/j.ymgme.2020.11.002</a>). *Co-first author.</p>
								<h2 class="minor">Genomic Software</h2>
								<span class="image main"><img src="images/EHdn_Titleshot.png" alt="" /></span>
								<p><b>ExpansionHunter De Novo (co-first)</b></p>
								<p>This is a tool for calling large repeat expansions in short-read PCR-free whole genome sequencing datasets. This software excels at identifying large expansions, including non-reference repeat expansion-insertions. The tool was created in collaboration with co-first authors Egor Dolzhenko (Illumina) and Mark Bennett (WEHI).</p>
								<p>Egor Dolzhenko*, Mark Benett*, <b>Phillip A Richmond*</b>, Brett Trost, Sai Chen, Joke JFA van Vugt, Charlotte Nguyen, Giuseppe Narzisi, Vladimir G Gainullin, Andrew M Gross, Bryan R Lajoie, Ryan J Taft, Wyeth W Wasserman, Stephen W Shcerer, Jan H Veldink, David R Bentley, Ryan KC Yuen***, Melanie Bahlo***, Michael A Eberle***. ”ExpansionHunter Denovo: A computational method for locating known and novel repeat expansions in short-read sequencing data”, Genome Biology, 21, 102 (2020). (online at <a href="https://doi.org/10.1186/s13059-020-02017-z">https://doi.org/10.1186/s13059-020-02017-z</a>). *Co-first author, ***Co- last author.</p>
								<span class="image main"><img src="images/FlexTyper_Titleshot.png" alt="" /></span>
								<p><b>FlexTyper (co-first)</b></p>
								<p>This paper was a journey, that started with the question: what if we could better query sequences from raw fastq files. With motivation in graph-based applications, population reference genome selection, and flexible reference-free analysis of short-read DNA sequencing data. Most of the coding done by Alice Kaye and Jacques Godfrain, most of the story telling, application, and algorithm development/testing by Alice Kaye and myself.</p>
								<p><b>Phillip A. Richmond*</b>, Alice M. Kaye*, Godfrain Jacques Kounkou, Tamar V. Av-Shalom, Wyeth W Wasserman. ”Demonstrating the utility of flexible sequence queries against indexed short reads with FlexTyper.” PLoS Computational Biology, (2021). (online at <a href="https://doi.org/10.1371/journal.pcbi.1008815">https://doi.org/10.1371/journal.pcbi.1008815</a>). *Co-first author.</p>
								<span class="image main"><img src="images/GeneBreaker_Titleshot.png" alt="" /></span>
								<p><b>GeneBreaker (co-first)</b></p>
								<p>GeneBreaker is a simulation framework for creating rare genetic disease scenarios. It enables simulation of diverse classes of variants, including novel user-created, or known pathogenic variants. The online webtool returns variants in VCF+PED format for downstream simulation and testing of automated analysis frameworks.</p>
								<p><b>Phillip A. Richmond*</b>, Tamar V. Av-Shalom*, Oriol Fornes, Bhavi Modi, Alison Elliott, Wyeth W. Wasserman. ”GeneBreaker: Variant simulation to improve the diagnosis of Mendelian rare genetic diseases.” Human Mutation, (2020). (online at <a href="https://doi.org/10.1002/humu.24163">https://doi.org/10.1002/humu.24163</a>). *Co-first author.</p>
								<h2 class="minor">Molecular Biology and Gene Regulation</h2>
								<span class="image main"><img src="images/Polyploidy_Titleshot.png" alt="" /></span>
								<p><b>Polyploidy can drive rapid adaptation in yeast</b></p>
								<p>This paper includes my undergraduate thesis work, where I examined the genotypic effects of directed evolution on a ploidy-series (1N, 2N, 4N) of saccharomyces cerevisiae. This was my first foray into genomics, and I was responsible for variant calling, and genome signal identification (including aneuploidy estimation), for the evolved strains.</p>
								<p>Anna M. Selmecki, Yosef E. Maruvka, <b>Phillip A. Richmond</b>, Marie Guillet, Noam Shoresh, Am- ber L. Sorenson, Subhajyoti De, Roy Kishony, Franziska Michor, Robin Dowell & David Pell- man. “Polyploidy can drive rapid adaptation in yeast”, Nature (519): 349-352, (2015). (online at <a href="https://doi.org/10.1038/nature14187">https://doi.org/10.1038/nature14187</a>).</p>
								<span class="image main"><img src="images/RIM101_Titleshot.png" alt="" /></span>
								<p>This remains one of my favourite papers, a detailed investigation into the regulation of a single yeast promoter, influenced by divergence in evolution, using classical and creative next-generation genetics techniques. I led the genome analysis and bioinformatics for this project.</p>
								<p>Timothy J Read, <b>Phillip A Richmond</b>, Robin D Dowell. (2016), “A trans-acting variant within the transcription factor RIM101 interacts with genetic background to determine its regulatory capac- ity”, PLoS Genetics 12(1): e1005746, (2016). (online at <a href="https://doi.org/10.1371/journal.pgen.1005746">https://doi.org/10.1371/journal.pgen.1005746</a>).</p>
							</article>

						<!-- Consulting -->
						<article id="consulting">
							<h2 class="major">Consulting</h2>
							<span class="image main"><img src="images/Sherlock_Cloudy.jpg" alt="" /></span>
							<p>Under construction, May 2022.</p>
							<p>Are you looking for input on your project, support managing your software stack, the need for pipeline development/optimization, or benchmarking of your analysis method? Get in touch! Ping me on the contact tab.</p>
							<h3>Consulting Experience</h3>
							<p>2021 - World Anti Doping Agency</p>
							<p>For a walk-through of this consulting experience please see this <a href="https://twitter.com/Phil_A_Richmond/status/1471550880260390919">twitter thread.</a> </p>

						</article>

						<!-- Contact -->
							<article id="contact">
								<h2 class="major">Contact</h2>
								<span class="image main"><img src="images/rosiesherlock.jpg" alt="" /></span>
								<p>You can reach me by email: phillip (dot) a (dot) richmond @ gmail.com</p>
								<p>Or get in touch with one of the links below!</p>
								<ul class="icons">
									<li><a href="https://twitter.com/Phil_A_Richmond" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
									<li><a href="https://www.facebook.com/phillip.richmond.7" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
									<li><a href="https://www.instagram.com/sherlock_holmes_doodle/" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
									<li><a href="https://github.com/Phillip-a-richmond" class="icon brands fa-github"><span class="label">GitHub</span></a></li>
								</ul>
							</article>

						<!-- Elements -->
							<article id="elements">
								<h2 class="major">Elements</h2>

								<section>
									<h3 class="major">Text</h3>
									<p>This is <b>bold</b> and this is <strong>strong</strong>. This is <i>italic</i> and this is <em>emphasized</em>.
									This is <sup>superscript</sup> text and this is <sub>subscript</sub> text.
									This is <u>underlined</u> and this is code: <code>for (;;) { ... }</code>. Finally, <a href="#">this is a link</a>.</p>
									<hr />
									<h2>Heading Level 2</h2>
									<h3>Heading Level 3</h3>
									<h4>Heading Level 4</h4>
									<h5>Heading Level 5</h5>
									<h6>Heading Level 6</h6>
									<hr />
									<h4>Blockquote</h4>
									<blockquote>Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan faucibus. Vestibulum ante ipsum primis in faucibus lorem ipsum dolor sit amet nullam adipiscing eu felis.</blockquote>
									<h4>Preformatted</h4>
									<pre><code>i = 0;

while (!deck.isInOrder()) {
    print 'Iteration ' + i;
    deck.shuffle();
    i++;
}

print 'It took ' + i + ' iterations to sort the deck.';</code></pre>
								</section>

								<section>
									<h3 class="major">Lists</h3>

									<h4>Unordered</h4>
									<ul>
										<li>Dolor pulvinar etiam.</li>
										<li>Sagittis adipiscing.</li>
										<li>Felis enim feugiat.</li>
									</ul>

									<h4>Alternate</h4>
									<ul class="alt">
										<li>Dolor pulvinar etiam.</li>
										<li>Sagittis adipiscing.</li>
										<li>Felis enim feugiat.</li>
									</ul>

									<h4>Ordered</h4>
									<ol>
										<li>Dolor pulvinar etiam.</li>
										<li>Etiam vel felis viverra.</li>
										<li>Felis enim feugiat.</li>
										<li>Dolor pulvinar etiam.</li>
										<li>Etiam vel felis lorem.</li>
										<li>Felis enim et feugiat.</li>
									</ol>
									<h4>Icons</h4>
									<ul class="icons">
										<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
										<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
										<li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
										<li><a href="#" class="icon brands fa-github"><span class="label">Github</span></a></li>
									</ul>

									<h4>Actions</h4>
									<ul class="actions">
										<li><a href="#" class="button primary">Default</a></li>
										<li><a href="#" class="button">Default</a></li>
									</ul>
									<ul class="actions stacked">
										<li><a href="#" class="button primary">Default</a></li>
										<li><a href="#" class="button">Default</a></li>
									</ul>
								</section>

								<section>
									<h3 class="major">Table</h3>
									<h4>Default</h4>
									<div class="table-wrapper">
										<table>
											<thead>
												<tr>
													<th>Name</th>
													<th>Description</th>
													<th>Price</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td>Item One</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Two</td>
													<td>Vis ac commodo adipiscing arcu aliquet.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Three</td>
													<td> Morbi faucibus arcu accumsan lorem.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Four</td>
													<td>Vitae integer tempus condimentum.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Five</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
											</tbody>
											<tfoot>
												<tr>
													<td colspan="2"></td>
													<td>100.00</td>
												</tr>
											</tfoot>
										</table>
									</div>

									<h4>Alternate</h4>
									<div class="table-wrapper">
										<table class="alt">
											<thead>
												<tr>
													<th>Name</th>
													<th>Description</th>
													<th>Price</th>
												</tr>
											</thead>
											<tbody>
												<tr>
													<td>Item One</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Two</td>
													<td>Vis ac commodo adipiscing arcu aliquet.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Three</td>
													<td> Morbi faucibus arcu accumsan lorem.</td>
													<td>29.99</td>
												</tr>
												<tr>
													<td>Item Four</td>
													<td>Vitae integer tempus condimentum.</td>
													<td>19.99</td>
												</tr>
												<tr>
													<td>Item Five</td>
													<td>Ante turpis integer aliquet porttitor.</td>
													<td>29.99</td>
												</tr>
											</tbody>
											<tfoot>
												<tr>
													<td colspan="2"></td>
													<td>100.00</td>
												</tr>
											</tfoot>
										</table>
									</div>
								</section>

								<section>
									<h3 class="major">Buttons</h3>
									<ul class="actions">
										<li><a href="#" class="button primary">Primary</a></li>
										<li><a href="#" class="button">Default</a></li>
									</ul>
									<ul class="actions">
										<li><a href="#" class="button">Default</a></li>
										<li><a href="#" class="button small">Small</a></li>
									</ul>
									<ul class="actions">
										<li><a href="#" class="button primary icon solid fa-download">Icon</a></li>
										<li><a href="#" class="button icon solid fa-download">Icon</a></li>
									</ul>
									<ul class="actions">
										<li><span class="button primary disabled">Disabled</span></li>
										<li><span class="button disabled">Disabled</span></li>
									</ul>
								</section>

								<section>
									<h3 class="major">Form</h3>
									<form method="post" action="#">
										<div class="fields">
											<div class="field half">
												<label for="demo-name">Name</label>
												<input type="text" name="demo-name" id="demo-name" value="" placeholder="Jane Doe" />
											</div>
											<div class="field half">
												<label for="demo-email">Email</label>
												<input type="email" name="demo-email" id="demo-email" value="" placeholder="jane@untitled.tld" />
											</div>
											<div class="field">
												<label for="demo-category">Category</label>
												<select name="demo-category" id="demo-category">
													<option value="">-</option>
													<option value="1">Manufacturing</option>
													<option value="1">Shipping</option>
													<option value="1">Administration</option>
													<option value="1">Human Resources</option>
												</select>
											</div>
											<div class="field half">
												<input type="radio" id="demo-priority-low" name="demo-priority" checked>
												<label for="demo-priority-low">Low</label>
											</div>
											<div class="field half">
												<input type="radio" id="demo-priority-high" name="demo-priority">
												<label for="demo-priority-high">High</label>
											</div>
											<div class="field half">
												<input type="checkbox" id="demo-copy" name="demo-copy">
												<label for="demo-copy">Email me a copy</label>
											</div>
											<div class="field half">
												<input type="checkbox" id="demo-human" name="demo-human" checked>
												<label for="demo-human">Not a robot</label>
											</div>
											<div class="field">
												<label for="demo-message">Message</label>
												<textarea name="demo-message" id="demo-message" placeholder="Enter your message" rows="6"></textarea>
											</div>
										</div>
										<ul class="actions">
											<li><input type="submit" value="Send Message" class="primary" /></li>
											<li><input type="reset" value="Reset" /></li>
										</ul>
									</form>
								</section>

							</article>

					</div>

				<!-- Footer -->
					<footer id="footer">
						<p class="copyright">&copy; Phillip Richmond, 2022. Design: <a href="https://html5up.net">HTML5 UP</a>.</p>
					</footer>

			</div>

		<!-- BG -->
			<div id="bg"></div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
